app/code/Magento/Catalog/Model/ResourceModel/Eav/Attribute.php

```diff
@@ -37,7 +37,6 @@ class Attribute extends \Magento\Eav\Model\Entity\Attribute implements
     const ENTITY = 'catalog_eav_attribute';
 
     const KEY_IS_GLOBAL = 'is_global';
-
     /**
```

Comment:
Revert this change)

Reply:
are you talking about the extra space in file? that i need to remove?

Reply:
@orlangur .i made the chnages.Please review this.
Thank You.

Reply:
Try `git checkout 2.3-develop -- app/code/Magento/Catalog/Model/ResourceModel/Eav/Attribute.php` to get this file in correct state.

Reply:
@orlangur file is in correct state now.I made the changes.